firefox-ios/Client/Frontend/Translations/LanguageDetector/LanguageDetector.swift

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import WebKit
+import NaturalLanguage
+
+/// A small utility for extracting a text sample from a web page and detecting its language.
+/// The sample is extracted using JS.
+final class LanguageDetector {
+    /// JS function that is called to get a page sample back. The function is implemented in `Summarizer.js`.
+    private static let languageSampleScript =
```

Comment:
do we need to make this a static instead of a let? So we don't need to reference by `Self.languageSampleScript` and can pass in `languageSampleScript` directly